src/definitions.ts

```diff
@@ -496,6 +496,12 @@ export class TSDBuilder extends ExportsWalker {
       sb.push(extendsNode.name.identifier.text); // TODO: fqn?
     }
     sb.push(" {\n");
+    if (!isInterface) {
+      indent(sb, this.indentLevel);
+      sb.push("static wrap(ptr: usize): ");
+      sb.push(name);
+      sb.push("\n");
```

Comment:
Looking at this again, is it missing a semicolon here?

Reply:
Indeed, missed it.